extensions-contrib/opencensus-extensions/src/test/java/org/apache/druid/data/input/opencensus/protobuf/OpenCensusProtobufReaderTest.java

```diff
@@ -325,6 +330,16 @@ public void testDimensionSpecExclusions() throws IOException
     Assert.assertFalse(row.getDimensions().contains("descriptor.color"));
   }
 
+  @Test
+  public void testInvalidProtobuf() {
+    reader = Mockito.mock(OpenCensusProtobufReader.class);
+    Mockito.when(reader.readAsList()).thenThrow(new ParseException("Protobuf message could not be parsed"));
+    Mockito.when(reader.read()).thenCallRealMethod();
```

Comment:
Nice use of mockito, but in this case it seems pretty simple to drive this behavior without mocking.

Can you use the same setup as the other test cases but put an invalid byte array instead?

```java
    byte[] invalidProtobuf = new byte[] { 0x00, 0x01 };
    ConsumerRecord consumerRecord = new ConsumerRecord(TOPIC, PARTITION, OFFSET, TS, TSTYPE,
        -1L, -1, -1, null, invalidProtobuf, HEADERS);
    KafkaRecordEntity kafkaRecordEntity = new KafkaRecordEntity(consumerRecord);
    // ...
```